streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorStateManagerTest.java

```diff
@@ -360,12 +339,11 @@ public void shouldRecycleStoreAndReregisterChangelog() {
 
         reset(context, store);
         context.uninitialize();
```

Comment:
I think you should remove this line. It was a verification in EasyMock and now with Mockito it is a call on the mock, which does not make a lot of sense. You make the call and in the next line you immediately verify that you made the call. Instead you should verify the call that is done within `stateMgr.registerStateStores(singletonList(store), context)`. Additionally to remove `context.uninitialize();` you need to move `verify(context).uninitialize();` below `stateMgr.registerStateStores(singletonList(store), context)`.

Reply:
Done.